Notes in RbConfig and in ruby --version an that we have debugger or
threadframe support.

diff --git a/tool/mkconfig.rb b/tool/mkconfig.rb
index f6f3351..3afcfc7 100755
--- a/tool/mkconfig.rb
+++ b/tool/mkconfig.rb
@@ -158,6 +158,7 @@ ARCH
 print "  universal = #{universal}\n" if universal
 print "  CONFIG = {}\n"
 print "  CONFIG[\"DESTDIR\"] = DESTDIR\n"
+print "  CONFIG[\"rb-threadframe\"] = \"0.3\"\n"
 
 versions = {}
 IO.foreach(File.join(srcdir, "version.h")) do |l|
diff --git a/version.h b/version.h
index b2b42e1..124b6a6 100644
--- a/version.h
+++ b/version.h
@@ -42,7 +42,7 @@
 # define RUBY_DESCRIPTION	    \
     "ruby "RUBY_VERSION		    \
     RUBY_PATCHLEVEL_STR             \
-    " ("RUBY_RELEASE_DATE	    \
+    " with debugger support ("RUBY_RELEASE_DATE	    \
     RUBY_REVISION_STR") "	    \
     "["RUBY_PLATFORM"]"
 # define RUBY_COPYRIGHT 	    \
